modular-code.md

```diff
@@ -11,36 +11,49 @@ Development does not want to spend the extra resources needed to develop modular
 * Time commitments might already have been made for customer deliveries (not changeable).
 
 **Forces:**
-* There is a learning curve to writing code that can be reused.
+* There is a learning curve to writing code that can be reused. Developers might not know how to write modular code. Education might be needed.
```

Comment:
> Education might be needed.

Feels like a potential solution rather than a force?

Reply:
Good catch (and education is indeed part of the solution). We can probably delete the sentence (Education might be needed).

Reply:
Removed this part:
> Education might be needed.

... as it was already part of the solution:
> * Offer education. Modular code is well-understood; there is a lot of literature in favor of this.